util/env/env.go

```diff
@@ -3,18 +3,63 @@ package env
 import (
 	"os"
 	"strings"
+	"github.com/stackanetes/kubernetes-entrypoint/logger"
 )
 
-func SplitEnvToList(env string, s ...string) (envList []string) {
-	separator := ","
-	if len(s) > 0 {
-		separator = s[0]
+type Dependency struct {
+	Name string
+	Namespace string
+}
+
+func SplitCommand() []string {
+    command := os.Getenv("COMMAND")
+	if command == "" {
+		return []string{}
 	}
+	commandList := strings.Split(command, " ")
+	return commandList
+}
+
+//SplitEnvToDeps returns list of namespaces and names pairs
+func SplitEnvToDeps(env string) (envList []Dependency) {
+	separator := ","
+
 	e := os.Getenv(env)
 	if e == "" {
 		return envList
 	}
 
-	envList = strings.Split(e, separator)
+	rawList := strings.Split(e, separator)
```

Comment:
@dkrzyszczyk `envList` or `envVars` is much better name for a list of environmental variables than `rawList` (saves me from checking how `rawList` is created)